### PR TITLE
feature: stepper component

### PR DESCRIPTION
**Pull request by csosamorales**
Considerations:
- I added the mint stepper as an example, it will change while we start to do de integrations (connect to wallet, connect to github, mint functionality)

### CODE REVIEW

**Review by zoruka — APPROVED** [review]

File: ui/src/components/layout/stepper/stepper.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { Button } from '../../core';
```
Comment (jsonsivar) [inline ui/src/components/layout/stepper/stepper.stories.tsx:1]: Is this file needed even if we don't want to do any storybook work for now?

**Review by jsonsivar — APPROVED** [review]
LGTM with just the question around the .stories file.  Also maybe we create issues or shortcut cards for the TODOs? I can take care of that.

File: ui/src/components/layout/stepper/stepper.stories.tsx (hunk 1 above)
Comment (csosamorales) [reply ui/src/components/layout/stepper/stepper.stories.tsx:1]: It's not mandatory, but it helps me to build just the component cause first I just did the stepper part, without all the related card and the tiltes and button, so it's helpful for the component building (specially the styling part)
